test/sys-info.ts

```diff
@@ -322,26 +332,61 @@ describe("SysInfo unit tests", () => {
 			});
 		});
 
-		describe("nativeScriptCliVersion", () => {
-			it("is null when tns is not installed", async () => {
-				childProcessResult.nativeScriptCliVersion = { shouldThrowError: true };
-				sysInfo = mockSysInfo(childProcessResult, { isWindows: false, isDarwin: true, dotNetVersion: "4.5.1" });
-				let result = await sysInfo.getSysInfo();
-				assert.deepEqual(result.nativeScriptCliVersion, null);
-			});
+		const testData: ICLIOutputVersionTestCase[] = [
+			{
+				testedProperty: "nativeScriptCliVersion",
+				method: (currentSysInfo: SysInfo) => currentSysInfo.getNativeScriptCliVersion()
+			},
+			{
+				testedProperty: "nativeScriptCloudVersion",
+				method: (currentSysInfo: SysInfo) => currentSysInfo.getNativeScriptCloudVersion()
+			}];
+
+		testData.forEach((testCase) => {
+			describe(testCase.testedProperty, () => {
+				it("is null when tns is not installed", async () => {
+					childProcessResult[testCase.testedProperty] = { shouldThrowError: true };
+					sysInfo = mockSysInfo(childProcessResult, { isWindows: false, isDarwin: true, dotNetVersion: "4.5.1" });
```

Comment:
`dotNetVersion`'s value could be declared as `const dotNetVersion = "4.5.1"` above and reused in all of these tests